src/components/GlobalLinterMessages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { getCodeLineAnchorID } from '../CodeView/utils';
+import { scrollToSelectedLine as _scrollToSelectedLine } from '../CodeView';
+
+type PublicProps = {
+  className?: string;
+  children: React.ReactNode[];
+  scrollToSelectedLine?: typeof _scrollToSelectedLine;
+};
+
+const GlobalLinterMessages = ({
+  className,
+  children,
+  scrollToSelectedLine,
```

Comment:
GlobalLinterMessages doesn't need to know about *what* the ref is used for. It's just a ref. Thus, the name of this prop should be something generic like `ref` or `containerRef`.